lib/mandrill-rails/web_hook_processor.rb

```diff
@@ -34,8 +34,8 @@ module Mandrill::Rails::WebHookProcessor
   extend ActiveSupport::Concern
 
   included do
-    skip_before_filter :verify_authenticity_token
-    before_filter :authenticate_mandrill_request!, :only => [:create]
+    skip_before_action :verify_authenticity_token
+    before_action :authenticate_mandrill_request!, :only => [:create]
```

Comment:
maybe is better to do something like ...

```ruby
    if respond_to?(:before_action)
      skip_before_action :verify_authenticity_token
      before_action :authenticate_mandrill_request!, :only => [:create]
    else
      skip_before_filter :verify_authenticity_token
      before_filter :authenticate_mandrill_request!, :only => [:create]
    end
```

So it works in older Rails versions.